src/Microsoft.AspNet.Authentication.OpenIdConnect/Events/RedirectFromIdentityProviderContext.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Http;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    public class RedirectFromIdentityProviderContext : BaseControlContext<OpenIdConnectAuthenticationOptions>
```

Comment:
You're right. Actually I think it was intended as a parallel to OAuthReturnEndpointContext for use in InvokeReplyPathAsync. It would fulfill https://github.com/aspnet/Security/issues/327

I'll just remove it for now.